Engine/src/Input/GLFWInputSystem.bf

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections;
+using SteelEngine.Events;
+using SteelEngine.Window;
+using glfw_beef;
+
+namespace SteelEngine.Input
+{
+	public class GLFWInputSystem
+	{
+		HashSet<int> _connectedGamepads = new HashSet<int>() ~ delete _;
+
+		public Result<void> Initialize()
```

Comment:
Why have a `Result<void>` return type if the only return value is `.Ok`?

Reply:
I feel like Initialize methods should be able to fail in case we create

Reply:
I will also add this to the initialize methods of the ECS